docs/UserGuide.md

```diff
@@ -229,16 +231,7 @@ Example of usage:
 `set r 500`
 
 Expected Outcome:
-```
-The resistor was set to 500.0 Ω
-	+---R-----C---+
-	|             |
-	|             |
-	+----+V_ac+---+
-Current Voltage: 0.0 V
-Total Resistance: 500.0 Ω
-Total Capacitance: 0.0 µF
-```
+![SetCircuit](diagrams/SetCircuitLabel.png)
```

Comment:
For this image could you change the green arrow to point at "the resistor was set to 500" instead of total resistance:
![image](https://user-images.githubusercontent.com/57522870/98458261-c740fc00-21c9-11eb-9fb9-0f8b6bdff116.png)

Reply:
Okay, changed!